### DISCUSSION TITLE
I don't see "Publish this action to the GitHub Marketplace" when creating a new version

### DISCUSSION THREAD

**Discussion by adamhenson** [community/community#24965] [Actions]
<p>Hello - I’ve searched around for a bit and I don’t see anyone with my issue exactly. <a href="https://github.com/foo-software/lighthouse-check-action" rel="nofollow noopener">I have a GitHub</a> action I’ve been working on tirelessly and I’m unable to cross the finish line (144 commits later). When following the instructions <a href="https://developer.github.com/marketplace/actions/publishing-an-action-in-the-github-marketplace/" rel="nofollow noopener">here</a>, I’m stuck at the beginning “Select  <strong>Publish this action to the GitHub Marketplace</strong>”. I was given access to GitHub Actions beta, however I don’t even see this option (screenshot attached). Do I need to do something to see this option?</p>
<p>Thanks!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/010ae89bc0b6c9e67d4cc58aae555f1bd9762452.png" data-download-href="/uploads/short-url/9dQDV9d4cxiuNhySygDPjZZTb4.png?dl=1" title="Screen Shot 2019-09-27 at 10.27.04 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/747743/181087918-d4f05852-6e87-43a0-ab0b-eb33f8a2c4f7.png" alt="Screen Shot 2019-09-27 at 10.27.04 AM.png" data-base62-sha1="9dQDV9d4cxiuNhySygDPjZZTb4" width="690" height="442" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/010ae89bc0b6c9e67d4cc58aae555f1bd9762452_2_690x442.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/010ae89bc0b6c9e67d4cc58aae555f1bd9762452_2_1035x663.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/010ae89bc0b6c9e67d4cc58aae555f1bd9762452_2_1380x884.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/010ae89bc0b6c9e67d4cc58aae555f1bd9762452_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-27 at 10.27.04 AM.png</span><span class="informations">2472×1586 344 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by adamhenson** [comment]
<p>Even when I clone <a href="https://github.com/actions/hello-world-javascript-action" rel="nofollow noopener">the example</a> by choosing “Use this template” I can reproduce. It seems like for some reason my account isn’t setup, even though it actually is (screenshot below). Wondering what more I need to do and why I can’t find documentation about it.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/317e2918785ba541b80921b14796975085cb5fb7.png" data-download-href="/uploads/short-url/73PH0JHvembSkjC5bhQVM1iKndB.png?dl=1" title="Screen Shot 2019-09-27 at 7.03.32 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/747743/181087919-2638db42-35c3-4c15-bc9f-6bc647d03906.png" alt="Screen Shot 2019-09-27 at 7.03.32 PM.png" data-base62-sha1="73PH0JHvembSkjC5bhQVM1iKndB" width="690" height="422" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/317e2918785ba541b80921b14796975085cb5fb7_2_690x422.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/317e2918785ba541b80921b14796975085cb5fb7_2_1035x633.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/317e2918785ba541b80921b14796975085cb5fb7_2_1380x844.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/317e2918785ba541b80921b14796975085cb5fb7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-27 at 7.03.32 PM.png</span><span class="informations">1524×934 82.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mscoutermarsh** [comment]
<p>Looking into this for you.</p>

**Comment by adamhenson** [comment]
<p><a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> - I have beta access already. See my screenshot in my second message.</p>

**Comment by adamhenson** [comment]
<p>Oh, I gotchya - my user account. Well that’s annoying and unintuitive. I do appreciate your help <a class="mention" href="/mscoutermarsh">@mscoutermarsh</a>. Fingers crossed - I just requested.</p>

**Comment by adamhenson** [comment]
<p>Haha, all good. Is there any way my request for access can be escalated?</p>

**Comment by mscoutermarsh** [comment]
<p>You’re in. No waitlist at the moment. :grinning:</p>

**Comment by adamhenson** [comment]
<p>Yes, I’m in - thank you so much <a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> </p>

**Comment by paramt** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2c4d9e79996b0fd0ffe0bf991648791deb8e298c.png" data-download-href="/uploads/short-url/6jVj14U4ToY8EnztQztznBCfALi.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/23563074/181087927-3c85258e-e777-4844-9caa-75c11491f62e.png" alt="image.png" data-base62-sha1="6jVj14U4ToY8EnztQztznBCfALi" width="690" height="420" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2c4d9e79996b0fd0ffe0bf991648791deb8e298c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">751×458 25.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’m having the same problem, even though it is enabled on my account already. It’s also enabled for my repository from the repo settings. Am I doing something wrong?</p>

**Comment by jwalton** [comment]
<p>I ran into this problem a few times because my <code>action.yml</code> file was accidentally named <code>action.yaml</code>.  :P  GitHub will only show a “Publish this action…” if it’s the first.</p>

**Reply by yogeshlonkar** [reply]
Just ran into this issue was not able to see these fields when releasing, `action.yml` works `action.yaml` doesn't work

**Comment by CedX** [comment]
<p>Same issue here: I can’t release my GitHub Action on the Marketplace because there isn’t any “publish” checkbox.<br>
See: <a href="https://github.com/cedx/setup-dart/releases" rel="nofollow noopener">https://github.com/cedx/setup-dart/releases</a></p>

**Comment by Deborah-Digges** [comment]
<p><a class="mention" href="/cedx">@cedx</a></p>
<p>Were you able to solve this? I am also not able to see the publish banner for my repository</p>

**Comment by CedX** [comment]
<p><a class="mention" href="/deborah-digges">@deborah-digges</a> Yes… by contacting the GitHub support (<a href="https://support.github.com/contact" rel="noopener nofollow ugc">https://support.github.com/contact</a>). The issue was on their side, not with my repository.</p>

**Comment by rayluo** [comment]
<p>Hi <span class="mention">@mscountermarsh</span> et al, I encountered same issue.</p>
<p>Can you help clarify whether my action repo would need to <a href="https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#about-publishing-actions" rel="noopener nofollow ugc"><em>use</em> yet another action</a>? If so, why?</p>
<p>UPDATE: Never mind, I tried and confirmed that, once I created a dummy workflow inside my Action repo, then I am allowed to publish my Action. It is still beyond me why such a dummy workflow would be needed.</p>

**Comment by Roms1383** [comment]
Same here I can see "Publish this Action to Marketplace" banner, but there's no checkbox to "Publish this action to the GitHub Marketplace" to be found when drafting the release. I contacted the support and hope for a quick answer.

**Reply by Roms1383** [reply]
I got in touch with the support, at the moment there's a minor issue the manifest must be named `action.yaml` they're aware of it and it should be fixed sooner or later.

**Comment by Relequestual** [comment]
I'm also experiencing this issue.
I created a support ticket. I'll update here for others if I find out a resolution beyond "contact support". Ugh.

**Comment by brett-james-rocketlab** [comment]
Still have this problem now. I click Draft a Release, and I only get as far as making a new release with no option to make it be on the marketplace.

**Comment by Armadillidiid** [comment]
Can someone please share their fix? It's 2024 and I can't believe this issue still persists.

**Reply by tealtony** [reply]
Check if your repo is listed as private. I was having this issue. Only public repos show with Add to Marketplace.
